scripts/build/Dockerfile.archlinux

```diff
@@ -0,0 +1,45 @@
+FROM archlinux
+ARG CC=gcc
+ARG ENV1=FOOBAR
+
+RUN pacman -Sy --noconfirm \
```

Comment:
The Arch wiki recommends using `-Syu` to avoid dependency issues:
https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported

Reply:
Fixed. Thanks!